### ISSUE TITLE
Path for am_tds_ctc_librispeech_dev_clean.bin 

### ISSUE THREAD

**Issue by ML6634** [facebookresearch/wav2letter#692]
TDS CTC training went through:

<pre>...
Falling back to using letters as targets for the unknown word: heuchera
Skipping unknown entry: &apos;heuchera&apos;
Falling back to using letters as targets for the unknown word: d&apos;avrigny
Skipping unknown entry: &apos;d&apos;avrigny&apos;
Falling back to using letters as targets for the unknown word: linnell&apos;s
Skipping unknown entry: &apos;linnell&apos;s&apos;
Falling back to using letters as targets for the unknown word: julian&apos;s
Skipping unknown entry: &apos;julian&apos;s&apos;
I0614 15:29:32.598239   202 Train.cpp:345] epoch:        1 | nupdates:         1501 | lr: 0.000310 | lrcriterion: 0.000000 | runtime: 00:21:56 | bch(ms): 877.29 | smp(ms): 0.49 | fwd(ms): 281.01 | crit-fwd(ms): 9.09 | bwd(ms): 369.81 | optim(ms): 223.06 | loss:   59.11739 | train-TER: 120.99 | train-WER: 114.98 | dev-clean-loss:   44.62415 | dev-clean-TER: 100.00 | dev-clean-WER: 100.00 | dev-other-loss:   42.02407 | dev-other-TER: 100.00 | dev-other-WER: 100.00 | avg-isz: 1227 | avg-tsz: 037 | max-tsz: 080 | hrs:    5.12 | thrpt(sec/sec): 14.00
Memory Manager Stats
MemoryManager type: CachingMemoryManager
Number of allocated blocks:448
Size of free block pool (small):48
Size of free block pool (large):48
Total native mallocs:219
Total native frees:118
I0614 15:30:06.312266   202 Train.cpp:748] Finished training
</pre>

To decode, I need to fill out the path for --am=[...]/am_tds_ctc_librispeech_dev_clean.bin in decode_tds_ctc_gcnn_clean.cfg. However, it sounds that I do not have the file: am_tds_ctc_librispeech_dev_clean.bin. Earlier I thought it should be --am=/root/w2l/saved_models/am_tds_ctc_librispeech/am_tds_ctc_librispeech_dev_clean.bin. Unfortunately, it was not correct. Any ideas? Thank you!

**Comment by tlikhomanenko** [comment]
We specified this name `am_tds_ctc_librispeech_dev_clean.bin` if you are downloading the pretrained model (it has this file name). In your case go to rundir and find there the model and specify its path.

About your training you need to fix `--iters=10000000` (we changed the behaviour of this flag from iters to number of updates, so your training did only 1500 updates).

**Comment by ML6634** [comment]
I am training it using --iter=10000000. It has taken over 10 hours, and it sounds that it is training smoothly.

I do not think that I have downloaded the pretrained model. What is the command to do that, or what is the link? Thank @tlikhomanenko for the help!

**Comment by tlikhomanenko** [comment]
Pre-trained models which you can download and use are listed on the page https://github.com/facebookresearch/wav2letter/tree/master/recipes/models/sota/2019.

**Comment by ML6634** [comment]
Thank @tlikhomanenko for the quick response!
